web/src/reducers/budget.test.js

```diff
@@ -363,10 +363,12 @@ describe('budget reducer', () => {
           keyPersonnel: [
             {
               costs: { '1931': 150, '1932': 151, '1933': 152 },
+              fte: { '1931': 1, '1932': 1, '1933': 1 },
               hasCosts: false
             },
             {
               costs: { '1931': 150, '1932': 1500, '1933': 15000 },
+              fte: { '1931': 1, '1932': 1, '1933': 1 },
```

Comment:
I know this is still a WIP but just wanted to go ahead and plant this seed: at least one of these FTE amounts should be something other than 1. Maybe one should be zero and another should be between 0 and 1. That will verify that key personnel costs are being adjusted by their FTE percentages.

Then Jerome or I can hand-calculate whether the number are right to verify the test result. 😅